.github/workflows/build.yml

```diff
@@ -30,13 +40,34 @@ jobs:
           path: '.'
           pattern: '*.sh'
 
+  tests:
+    if: ${{ (github.event.ref_type != 'tag') }}
+    needs: ['lint']
+    runs-on: ubuntu-18.04
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Build
+        run: ./ci/build.sh
+
+      - name: Running tests
+        run: ./ci/test.sh
+
   istgt:
     runs-on: ubuntu-latest
```

Comment:
The tests and actual build runs on 2 types of VMs. One is `ubuntu-latest` and the other is `ubuntu-18.04`. May need to fix on using one version

Reply:
Here in tests the code is build locally so it depends on the os, whereas in the istgt the code is build in a docker container build stage which has ubuntu-18 as the build stage image. https://github.com/openebs/istgt/blob/3a853b354eab46258f46331bdea4bdf4c7e549ab/docker/istgt.Dockerfile#L15